### PR TITLE
Scale test scenario update.

### PR DESCRIPTION
**Pull request by shahzebnp**
1. Scale test scenario update.
    2. Indicating Large Scale deployment page in sidebar

### CODE REVIEW

File: documentation/largescaledeployment.md [hunk 1]
```diff
@@ -3,7 +3,18 @@ title: Maximum scale supported by VCP
 ---
 **Maximum scale validated - 1000 nodes Kubernetes cluster**
 
-The following configuration is used to validate the scale limits for VCP.
+**Test Scenario**
+1. Creating a kubernetes cluster with 1000 worker nodes and enabling VCP on all of them
```
Comment (SandeepPissay) [inline documentation/largescaledeployment.md]: Creating -> Create 

Here and below.

File: documentation/largescaledeployment.md [hunk 2]
```diff
@@ -3,7 +3,18 @@ title: Maximum scale supported by VCP
...
+2. Creating a statefulset(nginx) with persistent volume dynamically provisioned where provisioner is VCP.
```
Comment (SandeepPissay) [inline documentation/largescaledeployment.md]: VCP -> vsphere-volume

File: documentation/largescaledeployment.md (hunk 1 above)
Comment (SandeepPissay) [inline documentation/largescaledeployment.md]: expand VCP here and use the abbreviation in other places. Reader would not know what VCP means the first time.

File: documentation/largescaledeployment.md [hunk 3]
```diff
@@ -3,7 +3,18 @@ title: Maximum scale supported by VCP
...
+   Kubernetes StatefulSets allows to us verify 4 key workflows in VCP.
+    a. Create vSphere persistent volume
+    b. Attach vSphere persistent volume to a pod
+    c. Detach  vSphere persistent volume from a pod
+    d. Delete vSphere persistent volume
+3. Once the StatefulSet is created with replicas: 1000,  we see 1000 pods created with vSphere persistent volume attached for each pod. Each pod has a vSphere persistent volume attached to it. Each pod is provisioned on an individual kubernetes worker node.
+4. The datastore used to create volumes is a shared VMFS datastore and disk type is thin.
```
Comment (SandeepPissay) [inline documentation/largescaledeployment.md]: My understanding is that the datastore used is NFS. Please confirm with Divyen.

Comment (shahzebnp) [reply documentation/largescaledeployment.md]: We used NFS based datastore to setup nodes beyond 1000; when we were creating around 2000 nodes.
Till 1000 shared VMFS worked fine.

**Review by divyenpatel — APPROVED** [review]
LGTM

File: documentation/_data/sidebars/home_sidebar.yml [hunk 4]
```diff
@@ -55,6 +55,9 @@ entries:
       - title: Best Practices
         url: /bestpractices.html
         output: web
+      - title: Large Scale Deployment
```
Comment (tusharnt) [inline documentation/_data/sidebars/home_sidebar.yml]: Should we rename it to "Maximum supported scale limit "?

**Review by tusharnt — CHANGES_REQUESTED** [review]

File: documentation/_data/sidebars/home_sidebar.yml (hunk 4 above)
Comment (shahzebnp) [reply documentation/_data/sidebars/home_sidebar.yml]: I see. Sure.

File: documentation/_data/sidebars/home_sidebar.yml [hunk 5]
```diff
@@ -55,6 +55,9 @@ entries:
       - title: Best Practices
         url: /bestpractices.html
         output: web
+      - title: Maximum supported scale limit
+        url: /largescaledeployment.html
```
Comment (tusharnt) [inline documentation/_data/sidebars/home_sidebar.yml]: Why  not rename the URL as well?

**Review by tusharnt — APPROVED** [review]

**Pull request merged**